packages/app/src/server/models/bookmark-folder.ts

```diff
@@ -73,7 +86,15 @@ bookmarkFolderSchema.statics.findFolderAndChildren = async function(
 
   const parentFolder = await this.findById(parentId) as unknown as BookmarkFolderDocument;
```

Comment:
I think mongoose find may be Null, but can we force determine the type?
How about return an error if it is null?

Reply:
I just defined parentFolder  type and add errorr exception if parentFolder is null
I also modified the query when parentId is null. 
When parentId it will load the root bookmark folders , otherwise it will load all child folders